tekton/release-cheat-sheet.md

```diff
@@ -130,6 +130,24 @@ the pipelines repo, a terminal window and a text editor.
 1. Update [the catalog repo](https://github.com/tektoncd/catalog) test infrastructure
 to use the new release by updating the `RELEASE_YAML` link in [e2e-tests.sh](https://github.com/tektoncd/catalog/blob/master/test/e2e-tests.sh).
 
+1. Update the docs on the website with the latest version released:
+
+    ```bash
+    # From the website repo root, clean and up to date
+    # Update the config
+    cd sync
+    python3 -m venv .venv
+    source .venv/bin/activate    
+    pip3 install -r requirements.txt
+    ./versions.py add -p pipeline [version]
+
+    # Stage, commit and push the change
+    git checkout -b pipeline_vXYZ_docs
```

Comment:
we could potentially use the `$TEKTON_VERSION` variable here and in the few strings below instead of `vXYZ` I think?

Reply:
Ah, but then we would also need to export the env vars maybe because we invoke virtualenv. Is that right?

Reply:
@sbwsg I don't have explanation but exporting `TEKTON_VERSION` in step 6 above is available even after invoking virtual env. (my python knowledge is rusted 😢 )

```
[63] -> export TEKTON_VERSION=v0.20.0

[64] -> source .venv/bin/activate
(.venv)

 [65] -> env | grep TEKTON
TEKTON_VERSION=v0.20.0
```